src/planner/plan/plan_port_db.cpp

```diff
@@ -26,15 +28,22 @@ std::unique_ptr<LogicalPlan> Planner::planExportDatabase(const BoundStatement& s
     auto fileTypeStr = FileTypeUtils::toString(fileType);
     StringUtils::toLower(fileTypeStr);
     auto copyToSuffix = "." + fileTypeStr;
+    auto functions = clientContext->getCatalog()->getFunctions(clientContext->getTx());
+    std::string name = common::stringFormat("COPY_{}", FileTypeUtils::toString(fileType));
+    auto func =
+        function::BuiltInFunctionsUtils::matchFunction(clientContext->getTx(), name, functions);
+    KU_ASSERT(func != nullptr);
+    auto copyFunc = *func->constPtrCast<function::CopyFunction>();
     for (auto& exportTableData : *exportData) {
         auto regularQuery = exportTableData.getRegularQuery();
         KU_ASSERT(regularQuery->getStatementType() == StatementType::QUERY);
         auto tablePlan = getBestPlan(*regularQuery);
-        auto copyTo = std::make_shared<LogicalCopyTo>(filePath + "/" + exportTableData.tableName +
-                                                          copyToSuffix,
-            fileType, exportTableData.columnNames, exportTableData.getColumnTypesRef(),
-            boundExportDatabase.getCopyOption(), tablePlan->getLastOperator(),
-            exportTableData.isParallel);
+        auto path = filePath + "/" + exportTableData.tableName + copyToSuffix;
+        function::CopyFuncBindInput bindInput{exportTableData.columnNames, std::move(path),
```

Comment:
It's fine to have it here. Though I would suggest try to do it in the binder while binding export database